user/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db          import models
+
+from .validation        import Validate_firstname,Validate_lastname,Validate_email,Validate_password
+
+class User(models.Model):
+    firstname           = models.CharField(
+        max_length      = 100,
+        validators      = [Validate_firstname]
+    )
```

Comment:
```suggestion
    firstname           = models.CharField(max_length = 100, validators = [Validate_firstname])
```
줄 분리의 기준은 가독성을 높이기 위함이며, 가독성이 높아야하는 이유는 필요한 로직에 집중하기 위해서입니다. 클래스에서, 특히 모델을 구성하는 클래스에서 중요한 것은 모델의 필드, 즉 클래스의 속성이이기 때문에, 속성의 부가 정보까지 분리하지 않으셔도 됩니다 
깔끔하게 만드시느라 고생하셨습니다 👍

Reply:
아래 속성들도 모두 수정해주세요~

Reply:
넵 감사합니다. 수정하였습니다.